ocs_ci/ocs/utils.py

```diff
@@ -975,8 +981,11 @@ def collect_ocs_logs(dir_name, ocp=True, ocs=True, mcg=False, status_failure=Tru
             os.path.expanduser(ocsci_config.RUN["log_dir"]),
             f"{dir_name}_{ocsci_config.RUN['run_id']}",
         )
-
-    if ocs:
+    # Restricting to one time log collection for both OCS&OCP in case of status failure
```

Comment:
@jimdevops19 you cannot implement the logic here as this will basically affect all tests and you will collect logs only on first one failed test and any other failed test. The point of the issue: 
https://github.com/red-hat-storage/ocs-ci/issues/5010
and issue:
https://github.com/red-hat-storage/ocs-ci/issues/4092

Is about that in case the deployment fail, it collect logs for that failed execution + also it also collect once again the logs at the end of the execution becasue collect logs on success.

So this should be fixed on those two places:
https://github.com/red-hat-storage/ocs-ci/blob/d4a9c7d42a5e7895bab580255374f33852c0bdf9/ocs_ci/deployment/deployment.py#L176
And here:
https://github.com/red-hat-storage/ocs-ci/blob/d353b25323bbf20e4298b6339f84c4be8888e33d/tests/conftest.py#L3559

Or you can keep somehow the logic here, but you will need to have extra parameter in this function like: skip_if_already_collected. and you will apply the logic of skipping that collection if we already have it collected.